index.html

```diff
@@ -129,9 +142,9 @@ <h1 id="about_us" class="sb-content-headline sb-spaced-headline">Über uns</h1>
             </p>
 
             <div class="sb-picture-item-big">
-              <a href="assets/images/2017/P1300802.JPG" data-lightbox="about_us_gallery">
```

Comment:
remove this, we doesn't need the group picture twice

Reply:
will do, was wondering about that, "about" looks a bit empty with just 2 paragraphs